src/server/handlers/store.js

```diff
@@ -16,6 +21,12 @@ export const registerName = (req, res) => {
     },
     body: JSON.stringify({ snap_name: snapName })
   }).then((response) => {
+    getMemcached().del(cacheId, (err) => {
+      if (err) {
+        logger.error(`Error deleting ${cacheId} from memcached:`, err);
+      }
+    });
+
```

Comment:
Do we perhaps want to move the responding code inside the `del` callback, so that we get deterministic behaviour?